R/model-tree.R

```diff
@@ -1,12 +1,15 @@
 #' Create a tree diagram of a modeling directory
 #'
-#' @param .log_df a `bbi_run_log_df` tibble (the output of `run_log()`) ***or***
+#' @param .log_df A `bbi_run_log_df` tibble (the output of `run_log()`) ***or***
 #'        a base directory to look in for models. See details for more options.
-#' @param include_info vector of columns present in `.log_df` to include in the
+#' @param include_info A vector of columns present in `.log_df` to include in the
 #'        tooltip.
-#' @param color_by a run log column to color the nodes by. Can be helpful for
+#' @param color_by A run log column to color the nodes by. Can be helpful for
 #'        identifying which models are starred, have heuristics, etc. See details
 #'        for more information.
+#' @param size_by A **numeric** (or integer) run log column to size the nodes by.
+#'        If not specified, the default sizing is `'leafCount'`, which sizes the
```

Comment:
nit-pick and okay to leave as is: I don't think adding the specific name of the collapsibleTree value here ("leaftCount") adds anything on top of the description of the behavior.